src/SlashPineTech.Forestry.ServiceModules/ServiceModuleNameAttribute.cs

```diff
@@ -1,11 +1,13 @@
 using System;
+using JetBrains.Annotations;
 
 namespace SlashPineTech.Forestry.ServiceModules;
 
 /// <summary>
 /// Describes a named implementation of a service module.
 /// </summary>
 [AttributeUsage(AttributeTargets.Class)]
+[MeansImplicitUse]
```

Comment:
This should result in polymorphic modules not being marked as unused due to not being statically referenced.